python/hail/docs/getting_started_developing.rst

```diff
@@ -0,0 +1,66 @@
+.. _sec-getting_started_developing:
+
+==========================
+Getting Started Developing
+==========================
+
+You'll need:
+
+- The `Java 8 JDK <http://www.oracle.com/technetwork/java/javase/downloads/index.html>`_.
+- `Spark 2.0.2 <http://spark.apache.org/downloads.html>`_
+- `Anaconda <https://www.continuum.io/downloads>`_.
+
+-------------------
+Building a Hail JAR
+-------------------
+
+See :ref:`building-hail-from-source`.
+
+-----------------
+Building the Docs
+-----------------
+
+Hail uses conda environments to manage python dependencies. First, create a
+conda environment for hail:
+
+.. code-block:: bash
+
+    conda env create hail -f ./python/hail/environment.yml
+
+And run the ``createDocs`` gradle task in the environment:
+
+.. code-block:: bash
+
+    (source activate hail && gradle createDocs) ; source deactivate hail
```

Comment:
Yes, I would suggest conda environments for users and developers all. Project-specific environments is effectively what gradle gives us for Java: each gradle project has all its particular dependencies downloaded and linked to it. There's no problem if two projects on the same computer want to use different versions of the Jackson JSON library.

The non-conda python community uses virtualenv, with which I'm more familiar, but since we already recommend conda, I went with conda environments instead. They both solve the same problem in the same way.

I know this isn't the Unix Way™, but it's the way the python community handles dependency management.